subprojects/parseq-http-client/src/main/java/com/linkedin/parseq/httpclient/HttpClient.java

```diff
@@ -15,9 +19,9 @@ public class HttpClient {
    * then new client is created with default configuration.
    * @return raw http client
    */
-  public static synchronized AsyncHttpClient getNingClient() {
+  public static synchronized AsyncHttpClient getAsyncHttpClient() {
     if (_client.get() == null) {
-      initialize(new AsyncHttpClientConfig.Builder().build());
+      initialize(new Builder().build());
     }
     return _client.get();
   }
```

Comment:
I think we should instead introduce `getAsyncHttpClient()` alongside `getNingClient()` and mark `getNingClient()` as `@Deprecated`. I know the type has changed, and there's nothing we can do about that, but there are users of this code that will break if `getNingClient()` disappears on a minor version bump.

Reply:
I need to confer with the team about the other breaking changes.

Reply:
The only concern I would have with that is if ning needs a different version of netty (or other) that the new project needs. What are your thoughts on that behavior? Would you like to see the dependency resolution for netty?

Reply:
[netty-dependencies.txt](https://github.com/linkedin/parseq/files/14040076/netty-dependencies.txt)